.github/workflows/main.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+  lint-test:
+    uses: ./.github/workflows/lint-test.yaml
+
+
+  generate-sha-tag:
+    if: github.ref == 'refs/heads/main'
```

Comment:
```suggestion
```
This needs to be removed so that `publish-static-preview` can use it

Reply:
True true, since we have the 'github.ref' != '..main' case 
99ca1f946a9bb88ecd54f04b8ddfae96fa0a2fdc